R/gdp_count_time_steps.R

```diff
@@ -0,0 +1,35 @@
+#'@title Get Geo Data Portal time step counter.
```

Comment:
I'm not sure if this should be in this package, but for really big GDP files, it's nice to have something that just looks through the file and pulls out the time steps so you can parse out the data more carefully later and have a fixed size array to write into... I'm not attached to this though.